rancher/dashboard
# A11y storybook test runner

### Summary
Fixes #9423

- Enabled accessibility testing with the test runner
- Fixed broken `Banner` components

### How to test
- Start storybook: `yarn storybook`
- Start accessibility test : ` yarn storybook-test` It will show all passed or failed accessibility test on cli

**Note:** if there's an error related with the Chromium browser when starting the accessibility test, it's because you're missing the default browsers installation from Playwright, so just run `npx playwright install` to install them. You'll only need to do this once on your local machine.

You can then run ` yarn storybook-test` to start the tests again and the error should not appear anymore.
